tests/Language/AbstractTranslationTestCase.php

```diff
@@ -56,7 +56,7 @@ abstract class AbstractTranslationTestCase extends TestCase
         FrenchTranslationTest::class  => 'fr',
         //        HungarianTranslationTest::class          => 'hu',
         IndonesianTranslationTest::class => 'id',
-        //        ItalianTranslationTest::class            => 'it',
+        ItalianTranslationTest::class    => 'it',
         JapaneseTranslationTest::class => 'ja',
```

Comment:
```suggestion
        JapaneseTranslationTest::class   => 'ja',
```

Reply:
**Do this:**

![Screenshot 2022-11-18 172243](https://user-images.githubusercontent.com/9530214/202720611-f873e8fb-8197-47d0-b61d-6778c8beec96.jpg)